pkg/tinyusb/hw/hw_esp32.c

```diff
@@ -39,3 +39,8 @@ int tinyusb_hw_init(void)
 
     return 0;
 }
+
+void phy_bbpll_en_usb(bool en)
+{
+    (void)en;
+}
```

Comment:
Turns out that's not enough to get USB working with e.g. `examples/gnrc_networking`, it just makes it so that it compiles.